### PR TITLE
add new tool rpcompletion v5.13.1

### PR DESCRIPTION
**Pull request by kenza12**
Hi,
This is a PR for adding a new tool **rpcompletion v5.13.1** : https://github.com/brsynth/rptools/tree/master/rptools/rpcompletion
Thank you.

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/rpcompletion/rpcompletion.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<tool id="rpcompletion" name="Complete Reactions" version="@TOOL_VERSION@" profile="21.09">
+    <description>From the output of RP2Paths and RetroPath2.0, generate SBML unique and complete (with cofactors) pathways with mono-component reactions</description>
```
Comment (bgruening) [inline tools/rpcompletion/rpcompletion.xml:2]: the description is too long, can you shorten it?

File: tools/rpcompletion/rpcompletion.xml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        '$sink'
+        '$rp2paths_compounds'
+        '$rp2paths_pathways'
+        completed_pathways
+        --upper_flux_bound '$adv.upper_flux_bound'
+        --lower_flux_bound '$adv.lower_flux_bound'
+        --max_subpaths_filter '$adv.max_subpaths_filter'
+    ]]></command>
+    <inputs>
+        <param name="rp2paths_pathways" type="data" format="csv" label="RP2paths pathways" />
```
Comment (bgruening) [inline tools/rpcompletion/rpcompletion.xml:25]: can this be a tabular as well?

File: tools/rpcompletion/rpcompletion.xml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        <param name="rp2paths_compounds" type="data" format="tsv" label="RP2paths compounds" />
```
Comment (bgruening) [inline tools/rpcompletion/rpcompletion.xml:26]: please tabular not tsv

File: tools/rpcompletion/rpcompletion.xml [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        <param name="rp2_pathways" type="data" format="csv" label="RetroPath2.0 metabolic network" />
+        <param name="sink" type="data" format="csv" label="Sink from SBML" />
```
Comment (bgruening) [inline tools/rpcompletion/rpcompletion.xml:28]: from SBML? where is the SBML?

File: tools/rpcompletion/rpcompletion.xml [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+Project Links
+---------------------
+* `GitHub <https://github.com/brsynth/rptools>`_
+
+Version
+----------
+
+5.13.1
+
+Authors
```
Comment (bgruening) [inline tools/rpcompletion/rpcompletion.xml:96]: you can use https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator and remove this here

File: tools/rpcompletion/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+categories:
+- Synthetic Biology
+description: Galaxy tool that reads the output of RP2paths, parses it, and generates
+  SBML for each complete (cofactors) and unique pathway.
+homepage_url: https://github.com/brsynth/rptools/releases
```
Comment (bgruening) [inline tools/rpcompletion/.shed.yml:5]: can you add the remote_repository_url?

**Review by bgruening — COMMENTED** [review]
Some comments inline, thanks!

**Comment by bgruening** [comment]
@kenza12 this tool was not in the tool suite, correct?

**Comment by kenza12** [comment]
> @kenza12 this tool was not in the tool suite, correct?

Yes true, I just added it here: https://github.com/galaxyproject/tools-iuc/pull/5005/

**Pull request closed without merging**